config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
It's correct.

Though, I think mayba we can have it just as "Português". We probably won't have a diferent translation for Portugal.